nwb_conversion_tools/datainterfaces/ecephys/basesortingextractorinterface.py

```diff
@@ -24,7 +19,7 @@ class BaseSortingExtractorInterface(BaseDataInterface, ABC):
     @classmethod
     def get_source_schema(cls):
         """Compile input schema for the SortingExtractor."""
-        return get_schema_from_method_signature(cls.SX.__init__)
+        return get_schema_from_method_signature(cls.__init__)
```

Comment:
The actual biggest change here, even though it's only 1 line. Changes all the method signatures from having to rely on spikeextractors to our own local interfaces for them